From 1e37cc3aa674c4b052f0655fef59a910fd7325cb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 16 Dec 2021 14:53:25 -0800
Subject: [PATCH 1/2] libinotifytools: Include limit.h for PATH_MAX

musl builds fail since the required header which defines PATH_MAX is
missing, its perhaps included indirectly via glibc based systems

Upstream-Status: Submitted [https://github.com/inotify-tools/inotify-tools/pull/153]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libinotifytools/src/inotifytools.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libinotifytools/src/inotifytools.c b/libinotifytools/src/inotifytools.c
index 8bdfdb6..902eac2 100644
--- a/libinotifytools/src/inotifytools.c
+++ b/libinotifytools/src/inotifytools.c
@@ -17,6 +17,7 @@
 #include "inotifytools_p.h"
 #include "stats.h"
 
+#include <limits.h>
 #include <string.h>
 #include <strings.h>
 #include <stdlib.h>
-- 
2.34.1

